tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -1058,6 +1058,15 @@ def test_log_null_param(self):
             self.store.log_param(run.info.run_id, param)
         assert exception_context.exception.error_code == ErrorCode.Name(BAD_REQUEST)
 
+    def test_log_param_max_length_value(self):
+        run = self._run_factory()
+        tkey = "blahmetric"
+        tval = "x" * 500
+        param = entities.Param(tkey, tval)
+        self.store.log_param(run.info.run_id, param)
+        run = self.store.get_run(run.info.run_id)
+        assert run.data.params[tkey] == str(tval)
+
```

Comment:
Can we add a test to ensure that logging a param with value longer than 500 throws an exception?

Reply:
sure